SSIDManager/module.info

```diff
@@ -0,0 +1,6 @@
+{
+    "title": "SSID Manager",
+    "description": "Manage SSID Pools for PineAP",
+    "version": "1.0",
+    "author": "noncenz"
```

Comment:
The module.info JSON requires an array of devices.
```
"author": "noncenz",
"devices": [
     "nano",
     "tetra"
],
```

If you want to have the module on both devices, for example.

Reply:
All fixed, thanks for the help!